source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -0,0 +1,68 @@
+title: Check the location of your ``.so`` files
+ref: check-location
+level: 4
+content: |
+  The |odbc-driver-name| files are named ``libmdbodbca.so`` and
+  ``libmdbodbcw.so``. After downloading the |odbc-driver-name|,
+  move them to your ``/usr/local/lib`` directory.
+
+  The file named ``libmdbodbca.so`` is the ANSI driver, and the
+  file named ``libmdbodbcw.so`` is the UNICODE driver. Specify
+  the desired driver in your ``odbc.ini`` file.
+---
+title: Create an ``odbc.ini`` file
+ref: create-odbc-ini
+level: 4
+content: |
+  The file ``/etc/odbc.ini`` defines your Data Source Names. Below is
+  an example ``odbc.ini`` file which is configured to use the
+  |odbc-driver-name| with a DSN called ``MongoDBODBC``. Replace all
+  necessary placeholder values with values for your system.
+
+  .. code-block:: none
+
+     [MongoDBODBC]
+     Description = ODBC for MongoDB BI Connector
+     Driver      = /usr/local/lib/libmdbodbca.so
+     Trace       = Off
+     TraceFile   = stderr
+     ReadOnly    = yes
+     SERVER      = <server address>
+     PORT        = 3307
+     USER        = <username>
+     PASSWORD    = <password>
+     DATABASE    = <dbname>
+---
+title: Install unixODBC
+ref: install-unixodbc
+level: 4
+content: |
+  Install unixODBC with your preferred package manager. Alternatively,
```

Comment:
Should we backtick unixODBC since we're advising to install via package manager?

Reply:
Done.